package.json

```diff
@@ -20,6 +20,7 @@
     "react-router": "^5.3.4",
     "react-router-dom": "^5.3.4",
     "react-scripts": "5.0.1",
+    "serve": "^14.2.1",
```

Comment:
We shouldn't have this package in the MC app. Remove this line, run command 'yarn' and see if you can still use serve